tests/tap.sh

```diff
@@ -121,6 +121,15 @@ for ((i=0;i<${#backends[@]};++i)); do
         continue
     fi
 
+    # grep to skip t204-7 for MAGMA (different layout used for E-vectors)
+    if [[ "$backend" = *"magma" ]] \
+            && [[ "$1" = "t20"[4-7]* ]] ; then
+        printf "ok $i0 # SKIP - backend uses different E-vector layout $1 $backend\n"
+        printf "ok $i1 # SKIP - backend uses different E-vector layout $1 $backend stdout\n"
+        printf "ok $i2 # SKIP - backend uses different E-vector layout $1 $backend stderr\n"
+        continue
+    fi
```

Comment:
I think that this is fine for now, but we'll want a better long term strategy #425